### PR TITLE
Adjust styling of Home and HowTo

### PR DESCRIPTION
**Pull request by timgrossmann**
- Change color of Footer to dark grey to avoid green color overflow
- Spacing between cards
- Styling of lists 
- Styling of ToC on HowTo Page => dynamic Flexbox
- Removed "Highlighted" from middle element of Setup Options
- Replaced "bad quality" image of visualise tech proficiency
- adjusted Card layout to be more readable
- Removed parallax scrolling (won't work when page is scaled down)
- Adjusted size of icons to have more spacing around

---

Let me know what you think 😊 

@anoff Ready for review

### CODE REVIEW

**Comment by anoff** [comment]
love how you managed to break the stock action.. Oo

File: web/content/feature/editor.md [hunk 1]
```diff
@@ -2,16 +2,20 @@
 title: devradar Editor
 ---
 
-Document and present skills for individuals and teams. The devradar editor allows you to:
+**Document and present skills for individuals and teams. The devradar editor allows you to:**
```
Comment (anoff) [inline web/content/feature/editor.md]: Don't think a single line in an element needs to be bold 🙄

File: web/content/feature/editor.md [hunk 2]
```diff
@@ -2,16 +2,20 @@
...
+* Define quadrant categories
+* Create radar blips via GUI
+* View all skills in a list
+* Search for specific skills
+* Import/export via `.toml` files
+* Share your devradar via URL
+
+---
+
+<br />
```
Comment (anoff) [inline web/content/feature/editor.md]: is this necessary?

File: web/data/homepage.yml [hunk 3]
```diff
@@ -5,10 +5,10 @@ about:
...
   content : "Manage your skills as an individual using the devradar editor or track competence management within your team by using both the editor and the devradar teams app."
   item:
-    - text: "manage skills via web UI"
-    - text: "view all skills in a list or radar view"
-    - text: "import/export via `.toml` files"
-    - text: "overlay your teams devradars and focus on the key skills"
+    - text: "Manage skills via web UI"
+    - text: "View all skills in a list or radar view"
+    - text: "Import/export via `.toml` files"
+    - text: "Overlay your teams devradars and focus on the key skills"
```
Comment (anoff) [inline web/data/homepage.yml]: 👍

File: web/themes/anoff-project-gen/layouts/index.html [hunk 4]
```diff
@@ -4,37 +4,37 @@
...
+      <img class="img-fluid h-100 w-100" src="{{`images/illustrations/overlay-top.png` | relURL}}" alt="bg-shape">
     </div>
     <div id="radar-trans">
         <img class="img-fluid" src="{{`images/illustrations/radar-transparent.png` | relURL}}" alt="bg-shape">
       </div>
     <div class="layer" id="l1">
     </div>
     <div class="layer" id="l2">
-        <img src="{{`images/illustrations/blip-cogs.png` | relURL}}" alt="radar blip">
+        <img src="{{`images/illustrations/blip-cogs_new.png` | relURL}}" alt="radar blip">
```
Comment (anoff) [inline web/themes/anoff-project-gen/layouts/index.html]: new icons look better! :) No need to `-new` them just replace the old ones please

File: web/content/feature/editor.md (hunk 1 above)
Comment (timgrossmann) [reply web/content/feature/editor.md]: Just thought it looks nicer, should I remove it?

The text gets quite lost between the big heading and the gif, that was the idea 😁

File: web/content/feature/editor.md (hunk 2 above)
Comment (timgrossmann) [reply web/content/feature/editor.md]: Having no line break moves the "video" gif right to the horizontal line.
Do you want to have it removed? (Or done with CSS)

Comment (anoff) [reply web/content/feature/editor.md]: Ok, necessary then :D

File: web/content/feature/editor.md (hunk 1 above)
Comment (anoff) [reply web/content/feature/editor.md]: remove :)

File: web/themes/anoff-project-gen/layouts/index.html (hunk 4 above)
Comment (timgrossmann) [reply web/themes/anoff-project-gen/layouts/index.html]: Done 😉

File: web/content/feature/editor.md (hunk 1 above)
Comment (timgrossmann) [reply web/content/feature/editor.md]: Done

**Comment by timgrossmann** [comment]
@anoff Still breaking the CI 😄 
How to fix that

**Comment by anoff** [comment]
> @anoff Still breaking the CI 😄
> How to fix that

seems to be an issue with the github workflow for auto-labeling PRs. https://github.com/actions/labeler/issues/12
if it does not work from forks there is little use in it at all.. :(

**Comment by anoff** [comment]
Oh btw, I forgot to check back then; can you provide the origin of the updated use case image so I can check the license 😉

**Comment by timgrossmann** [comment]
@anoff absolutely.
https://unsplash.com/photos/CZ9AjMGKIFI

**Pull request merged**